nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -1055,6 +1055,36 @@ Superuser created successfully.
           </listitem>
         </itemizedlist>
       </listitem>
+      <listitem>
+        <para>
+          NSS modules are now globally provided (by a
+          <literal>/run/nss-modules</literal> symlink), similar to how
+          we handle OpenGL drivers.
+        </para>
+        <para>
+          This removes the need for nscd as a proxy for all NSS
+          requests, and avoids DNS requests leaking across network
+          namespaces.
+        </para>
+        <para>
+          While doing this upgrade, existing applications need to be
+          restarted, so they know how to pick up NSS modules from
+          <literal>/run/nss-modules/lib</literal>.
+        </para>
+        <para>
+          If you want to defer application restart to a later time,
+          explicitly enable <literal>nscd</literal> via
+          <literal>services.nscd.enable</literal> until the application
+          restarts.
```

Comment:
```suggestion
          restarts. This might also affect older application from earlier nixpkgs revisions that have been compiled with an older glibc.
```